audit/auditLog.json

```diff
@@ -209,6 +216,13 @@
       "auditorGitHandle": "sujithsomraaj",
       "auditReportPath": "./audit/reports/2025.04.15_Permit2Proxy(v1.0.3).pdf",
       "auditCommitHash": "5b212bd302a7ec0c027c51753d6d5a9937f6769e"
+    },
+    "audit20250422": {
+      "auditCompletedOn": "22.04.2025",
+      "auditedBy": "Sujith Somraaj (individual security researcher)",
+      "auditorGitHandle": "sujithsomraaj",
+      "auditReportPath": "./audit/reports/2025.04.22_LiFiDEXAggregator(v1.8.0).pdf",
```

Comment:
Should be `./audit/reports/2025.04.22_LiFiDexAggregator(v1.8.0).pdf`

Reply:
updated!